5-infrastructure/modules/bastion/README.md

```diff
@@ -0,0 +1,97 @@
+# Bastion
+
+## Purpose
+
+Create a Bastion VM using the `terraform-google-modules/bastion-host` module. All values for the module are passed as variables with one exception; the network tag is set to "bastion"
+
+> Reference: [terraform-google-bastion-host](https://github.com/terraform-google-modules/terraform-google-bastion-host)
+
+## Unit Tests
+
+```
+make usc1-a # Create bastion
+make destroy-usc1-a # Destroy bastion
+```
+
+> You will need to update the tfvars files to ensure the project id is set to a project you can access.
+
+### Dependencies
+
+The following terraform workspaces *must* exist:
+
+* central1
+
+## Calling
```

Comment:
nit: no need for module usage instruction since this is a wrapper around upstream module

Reply:
Removed in 795f245c193e34c5cc48c7ee516910b80e3f09c2